src/main/java/org/ow2/proactive/procci/model/occi/platform/Status.java

```diff
@@ -0,0 +1,28 @@
+package org.ow2.proactive.procci.model.occi.platform;
+
+import org.ow2.proactive.procci.model.exception.SyntaxException;
+
+public enum Status {
+    ACTIVE("The component is running"),
+    INACTIVE("The component is not running"),
+    ERROR("Error : the component is not working correctly");
+
+    private String message;
+
+    Status(String message) {
+        this.message = message;
+    }
+
+
+    public static Status getStatusFromString(String value) throws SyntaxException {
```

Comment:
Could it be replaced by Something like this =>   try { return Status.valueOf (value); } catch (IllegalArgumentException e){ ... }  .